pkgs/top-level/ocaml-packages.nix

```diff
@@ -47,8 +48,6 @@ let
 
     atdgen = callPackage ../development/ocaml-modules/atdgen { };
 
-    base = callPackage ../development/ocaml-modules/base { };
```

Comment:
The commit message might be misleading but these packages are not removed, the duplicated definition is removed but it will be transparent for users of these packages (since e.g. `ocamlPackages.janeStreet.base` is reexported as `ocamlPackages.base`).